requirements.txt

```diff
@@ -25,6 +27,7 @@ requests==2.25.1
 sentry-sdk==1.0.0
 six==1.15.0
 sqlparse==0.4.1
+typing-extensions==3.10.0.0
```

Comment:
what is `typing-extensions` used for?  don't we only need `email-validator` to validate an email?

Reply:
this is not something thats required for email-validator, I think this was added via VSCode, im not sure. But as i looked up for it, it says "Typing Extensions – Backported and Experimental Type Hints for Python"